bin/argos

```diff
@@ -1,2 +1,4 @@
 #!/usr/bin/env node
+
+process.env.NODE_ENV = 'production'
```

Comment:
What about the log message of the exit code status? I display it in dev but not in production. What about the console output that we output in dev & productino but not in the tests?